src/ui/templates/insurance/your-business/credit-control.njk

```diff
@@ -37,26 +39,28 @@
           fieldId: FIELD_ID,
           legendText: CONTENT_STRINGS.PAGE_TITLE,
           hintText: FIELD_HINT,
-          submittedAnswer: submittedValues[FIELD_ID],
+          submittedAnswer: application[FIELD_ID],
```

Comment:
should this not be application[FIELD_ID] or submittedValues[FIELD_ID] if validation errors?

Reply:
There's only 1 field so if there are validation errors there will not be any values available to populate.